### ISSUE TITLE
FDSNEvq does not return catalogs

### ISSUE THREAD

**Issue by mdenolle** [jpjones76/SeisIO.jl#16]
Hi,

I am trying to download the earthquake catalog from SoCal. Here is what i do:
using SeisIO.Quake
S =  FDSNevq("2018-06-01",reg=[32.0,38.0,-120.0,-115.0,-50.0,50.0],mag=[2.0,8.0],evw=[0.,375*24*3600.0],src="USGS")

It only returns one event (the same even for multiple arguments i entered):
(SeisHdr[    ID: quakeml
   INT:  0
   LOC: 33.4932 N, -116.49 E, 14.07 km
   MAG: ml 2.01 (g 0.0°, n 73)
    OT: 2018-06-02T20:45:04.18
   SRC:                                                                                     
   TYP: earthquake                                                                          
  MISC: 1 items
 NOTES: 0 entries
], SeisSrc[m₀ = 0.0; S = Float64[]; NP = Array{Float64}(0,0); PAX = Array{Float64}(0,0)])
It also does not like the SCEDC server, although I know it is listed there:
julia> S =  FDSNevq("2018-06-01",reg=[32.0,38.0,-120.0,-115.0,-50.0,50.0],mag=[2.0,8.0],evw=[0.,375*24*3600.0],src="SCEDC")
┌ Warning: Request failed
│ FDSN event query:
│ RESPONSE = 500 (Internal Server Error)
│ 
│ HTTP response is in misc["data"]
└ @ SeisIO ~/.julia/packages/SeisIO/UXhdy/src/Web/0_essentials.jl:16
┌ Warning: Catalog only contains 0 events (original request was 1)
└ @ SeisIO.Quake ~/.julia/packages/SeisIO/UXhdy/src/Quake/Web/FDSN.jl:99
(SeisHdr[], SeisSrc[])

**Comment by jpjones76** [comment]
I'm really sorry. Here's what's going on, and some questions about what you'd like to change:

- I need to update the help text for FDSNevq. FDSNevq in SeisIO v0.3.0 should be invoked with a tuple of output arguments; one for header data, one for source process info. So the syntax I have in the help text is outdated; it should be `(H,R) = FDSNevq(ot)` to get event headers in H and source process info in R. 

- By default, unless one specifies a number of events with keyword "nev", FDSNevq returns the nearest event to the OT that fits the other query parameters. I think I should actually change this behavior to return _all_ events by default. So, which sounds more usable to you: get one "best matching" event by default, or get all events by default? (As I think about this, returning all events seems more intuitive to me)

- I'm still tracking down what's going on with SCEDC. "Internal server error" usually means a bad request but there's nothing I see in the URL syntax that's a mistake. The same URL works with the USGS server. If I try the equivalent URL in my browser, the request times out.

**Comment by jpjones76** [comment]
Progress update:

- I changed the default behavior of FDSNevq tonight; it now returns full catalogs by default.
- Help text was updated for FDSNevq.
- I can only conclude that the SCEDC is doing something unusual with their FDSN event server configuration. I have no idea what that might be; I'll email them to ask tomorrow. The sample queries on their website work through the web interface, but nothing else works. Both SeisIO and custom browser queries fail; SeisIO gets an internal server error, browser queries stall. 
  - For example, suppose we try your query command with the SCEDC webpage.
  - Let's modify it to be https://service.scedc.caltech.edu/fdsnws/event/1/query?minlat=32.0&maxlat=38.0&minlon=-120.0&maxlon=-115.0&starttime=2018-06-01T00:00:00&endtime=2019-07-16T00:00:00&minmag=2.0&maxmag=8.0&format=text 
  - This times out forever...? Oh, wait, it finally works. It took half an hour just to return a text summary of events. I can't get it to return XML; it stalls forever.
  - The same query parameters work perfectly with the USGS server via FDSNevq in SeisIO, and 5420 events are found. The command required ~3 seconds.
